### ISSUE TITLE
JUnit output not working with CircleCI expanded insights

### ISSUE THREAD

**Issue by OmriBromberg** [datreeio/datree#640]
Steps to reproduce:
1. add a "run datree" step to a circleci job as follows (in our case its to verify against argo templates):
```
- run:
    name: run datree
    command: |
      curl https://get.datree.io | /bin/bash
      mkdir -p ~/test-results/datree
      datree test -p Argo templates/*.yaml --verbose -o JUnit > ~/test-results/datree/templates.xml
      datree test -p Argo cron/*.yaml --verbose -o JUnit > ~/test-results/datree/cron.xml
```

2. add "store_test_results" step as follows:
``` 
- store_test_results:
     path: ~/test-results/datree/
```

expected: seeing test results in `tests` in CircleCI
actually: ![image](https://user-images.githubusercontent.com/2093542/170000492-d75d46a2-46c0-4297-a2ce-40569d5aa857.png)

this guide from CircleCI was used to initiate this integration: https://circleci.com/blog/how-to-output-junit-tests-through-circleci-2-0-for-expanded-insights/

just as a verification effort here are some screenshots of the pipeline:
![image](https://user-images.githubusercontent.com/2093542/170000832-904d376c-47a6-43c1-81cd-5b957090d3db.png)
![image](https://user-images.githubusercontent.com/2093542/170000844-0f0a5bfd-b355-40eb-95bc-f8d5c1a71d27.png)

**Comment by alexfedin** [comment]
Thanks for submitting the issue 🙏 , we are looking into it

**Comment by alexfedin** [comment]
@OmriBromberg we found the issue, a fix will be coming soon.

btw, the issue is when all tests are passing the junit output is missing testcase tags so CircleCI doesn't see that any tests were run and shows 0.
if the tests are failing it works as expected


![Screen Shot 2022-05-24 at 18 47 47](https://user-images.githubusercontent.com/8144227/170078090-f552afd6-2be7-41d5-a36f-47edec07f13a.png)

**Comment by royhadad** [comment]
@OmriBromberg
fixed in the latest release (1.4.32), now the JUnit output also includes files that pass the policy check

![Screen Shot 2022-05-26 at 18 33 40](https://user-images.githubusercontent.com/39004075/170521920-568ff0cc-ea3b-45a9-9b5b-c119fbc2c542.png)
